src/librustc_session/options.rs

```diff
@@ -825,6 +828,8 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
     dep_tasks: bool = (false, parse_bool, [UNTRACKED],
         "print tasks that execute and the color their dep node gets (requires debug build) \
         (default: no)"),
+    disable_unique_symbols: bool = (false, parse_bool, [UNTRACKED],
```

Comment:
```suggestion
    trim_diagnostic_paths: bool = (true, parse_bool, [UNTRACKED],
```
https://github.com/rust-lang/rust/pull/73996#discussion_r471036398